outputs.tf

```diff
@@ -2,7 +2,22 @@
 # Outputs
 ########################################################################################################################
 
-#output "myoutput" {
-#  description = "Description of my output"
-#  value       = "value"
-#}
+output "api_endpoint" {
+  description = "API endpoint."
+  value       = "${var.region}.security-compliance-secure.cloud.ibm.com"
```

Comment:
This is hard coded to the public endpoint. Can you not get this value from the instance (or data lookup), as it will be different for private only instance